CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+Contributors Guide
+==================
+[__Quick Start__](README.md#quick-start) |
+[__FAQ__](docs/pages/faq.md) |
+[__Javadocs__](http://mesosphere.github.io/dcos-commons/reference/api/) |
+[__Contributing__](CONTRIBUTING.md) |
+[__Slack__](http://chat.dcos.io)
+
+Discuss contributions to Mesosphere USI using GitHub [issues](https://github.com/mesosphere/usi/issues) and make contributions with GitHub [pull requests](https://github.com/mesosphere/usi/pulls).
+
+### Bugs / Feature Requests
+Think you’ve found a bug? Want to see a new feature? Please open a case in our issue management tool, JIRA:
+* Login to the [DC/OS USI Public JIRA](https://jira.mesosphere.com/projects/DCOS_OSS_USI/issues).  You will need a Github or Google account to use this service.
```

Comment:
i thought there was recent discussion on making everything under either `DCOS_OSS` or `DCOS` only. I don't know if we would be able to make https://jira.mesosphere.com/projects/DCOS_OSS_USI/issues a valid URL.

Reply:
@meichstedt I guess we'll have a public facing JIRA here?

Reply:
Yeah, inclination towards just DCOS_OSS tickets is my understanding